source/textInfos/__init__.py

```diff
@@ -520,16 +540,19 @@ def getTextInChunks(self, unit):
 		@rtype: generator of str
 		"""
 		unitInfo=self.copy()
-		unitInfo.collapse()
-		while unitInfo.compareEndPoints(self,"startToEnd")<0:
+		unitInfo.end = unitInfo.start

```

Comment:
hmm interesting... I actually think in this case (both sides are `unitInfo`) the `collapse` method is clearer. If they were different objects, then using assignment would be clearer.